apis/Google.Cloud.Translation.V2/Google.Cloud.Translation.V2/TranslationClientImpl.cs

```diff
@@ -132,12 +132,19 @@ public override IList<TranslationResult> TranslateHtml(IEnumerable<string> htmlI
         }
 
         /// <inheritdoc />
-        public override IList<Detection> DetectLanguage(string text)
+        public override Detection DetectLanguage(string text) =>
```

Comment:
This implementation could be in the abstract base class, as it's just delegating.